CONTRIBUTING.md

```diff
@@ -32,6 +33,9 @@
 - Packages have to be run individually (eg: `cd packages/components` → `pnpm dev` to start development server)
 - Run `pnpm verify` at the root directory periodically, particularly, before pushing changes when a pull request is already opened.
 - Every branch should be associated with a PR.
+- Nearly all styles (colors, fonts, sizes etc.) are defined by our design team and provided for usage in our code via tokens (`packages/tokens/src/token.json`).
+  Components should use this tokens instead of individual styles as much as possible.
```

Comment:
```suggestion
  Components should use this tokens instead of individual styles as much as possible. Only where the tokens do not provide a styling, component specific styles should be added inside the [component-name].ts file
```